src/targets/node/terminalNodeLauncher.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AnyLaunchConfiguration, INodeTerminalConfiguration } from '../../configuration';
+import * as vscode from 'vscode';
```

Comment:
This is tying us to a model where the DA has to run in the extension host. I am still not sure about that, I think we should bring it up with Andre. Can leave it for now.